libs/dapp-config/src/currencies/currency-config.ts

```diff
@@ -250,7 +255,7 @@ export const currenciesConfig: Record<number, CurrencyConfig> = {
   },
   [CurrencyId.moonDEV]: {
     name: 'moonbase Dev',
-    symbol: 'moonDEV',
+    symbol: 'DEV',
```

Comment:
We already have the `DEV` token (in local evm), why do you want to update this?

Reply:
When `wallet_addEthereumChain()` is called to add `Moonbase Alpha`, `Metamask` throws error which says `nativeCurrency.symbol` can have `2-6` character string. `moonDEV` is 7. My bad didn't notice we were already using `DEV`.

Reply:
@devpavan04 you will still need to update this value to something unique (e.g. `MDEV`)